### DISCUSSION TITLE
Hotkey

### DISCUSSION THREAD

**Discussion by ClandestineClout** [gildas-lormeau/SingleFile#913] [General]
Since my CPU can't handle saving every page I'm forced to be more selective. There's an extension called Save Page Offline that has a keyboard shortcut Alt+S that is really convenient. How would we go about implementing something like that? I was tempted to just use their extension but they use .mhtml files and as far as I understand the SingleFile html page is superior in a number of ways (please correct me if I'm mistaken).

**Comment by gildas-lormeau** [comment]
There is already a shortcut in SingleFile to save the current tab, it's `Ctrl-Shit-Y`. The procedure to change it depends on the browser you're using.

**Comment by gildas-lormeau** [comment]
I'm moving this issue in the Discussion tab to keep it visible to everyone.

**Comment by x2elite** [comment]
Sorry, I don't want to open a new thread.
In 99% of my save actions I mark only a part of the page and use the submenu line (In german "Ausschnitt") for saving this part.
Is there a shortcut or hotkey for this? It would spare me a lot of mouse movements and clicks.
Thanks.
--
Ok, I found the information under "Additional notes", it says Ctrl-Shift-Y and it works.
--
